uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/b2c/IdLabB2cSisoPolicyPromptHandler.java

```diff
@@ -78,14 +77,16 @@ public void handlePrompt(@NonNull final String username, @NonNull final String p
 
     protected static IOAuth2LoginComponentHandler getAppropriateLoginComponentHandler(@NonNull final B2CPromptHandlerParameters parameters) {
         Logger.i(TAG, "Get Appropriate Login Component Handler..");
-        switch (parameters.getB2cProvider().getProviderName()) {
-            case LabConstants.B2CProvider.LOCAL:
+
+        switch (com.microsoft.identity.labapi.utilities.constants.B2CProvider.valueOf(
```

Comment:
same as above (can we directly import com.microsoft.identity.labapi.utilities.constants.B2CProvider instead?)

Reply:
Here, the B2CProvider from the local package is also being used in this class, so we'd need to rename one of them.

@shahzaibj, thoughts? On this and comment above this one.

Reply:
Let's rename this class (the one in ui/automation/interaction/b2c)

Reply:
Renamed to B2CWrapper